### PR TITLE
C29092: Possible typo box/toolbox

### PR DESCRIPTION
**Pull request by v-maudel**
Hello, @mikejo5000,
Translator has reported possible source content issue. 
"Linguist comment: "box" is unclear in this context. Could it be Toolbox?"
Please review and merge the suggested proposed file change to avoid this error. If you make related fix in another PR, then share your PR number, so we can confirm and close this PR.
Many thanks in advance.

### CODE REVIEW

**Comment by PRMerger8** [comment]
@v-maudel : Thanks for your contribution! The author, @mikejo5000, has been notified to review your proposed change.

File: docs/javascript/publish-nodejs-app-azure.md [hunk 1]
```diff
@@ -146,7 +146,7 @@ how to start the Node.js process. Details of the implementation can be seen at
 * You can connect to the running container via SSH for investigations. This is easily done using the Azure Portal. Select the App Service, and scroll down
 the list of tools until reaching **SSH** under the **Development Tools** section.
 * To aid in troubleshooting, go to the **Diagnostics logs** settings for the App Service, and change the **Docker Container logging** setting
-from **Off** to **File System**. Logs are created in the container under */home/LogFiles/*_docker.log*, and can be accessed on the box using SSH or FTP(S).
+from **Off** to **File System**. Logs are created in the container under */home/LogFiles/*_docker.log*, and can be accessed on the toolbox using SSH or FTP(S).
```
Comment (gewarren) [inline docs/javascript/publish-nodejs-app-azure.md]: I don't think this is correct. I think box might refer to "computer" or "machine".

Comment (v-maudel) [reply docs/javascript/publish-nodejs-app-azure.md]: Thanks @gewarren , if this is not "toolbox", I will need a confirmation so the linguist can translate it accordingly. 
Please, let me know if we can use computer instead. 
@Mikejo5000

**Comment by v-maudel** [comment]
Thanks @gewarren , if this is not "toolbox", I will need a confirmation so the linguist can translate it accordingly.
Please, let me know if we can use computer instead.
@Mikejo5000

**Comment by Mikejo5000** [comment]
Computer or machine is good. I think computer is preferred.

**Comment by v-maudel** [comment]
Thanks @Mikejo5000  and @gewarren

**Pull request closed without merging**